catalog/admin/module/catalog-admin-module-sources/src/main/webapp/js/model/Source.js

```diff
@@ -35,6 +35,8 @@ define([
             idAttribute: 'name',
             initialize: function () {
                 this.set('currentConfiguration', undefined);
+                this.set('currentUrl', undefined);
```

Comment:
Yeah I wasn't sure so I just added to what was already there. I can give it a try though and see if it will work